kommunicate/src/main/java/com/applozic/mobicommons/commons/core/utils/Utils.java

```diff
@@ -339,7 +339,7 @@ public static void writeToFile(Context context, String log) {
             BufferedWriter bufferedWriter = null;
             try {
                 String folder = "/" + Utils.getMetaDataValue(context, "main_folder_name");
-                File dir = new File(Environment.getExternalStorageDirectory().getAbsolutePath() + folder);
+                File dir = new File(context.getFilesDir().getAbsolutePath() + folder);
```

Comment:
Why we are changing  from `Environment.getExternalStorageDirectory()` to `context.getFilesDir()`

Reply:
`Environment.getExternalStorageDirectory()` is deprecated and will fail if external SD card is not present. 
Using `context.getFilesDir() `we will be logging to the correct directory of the application.

Reply:
![image](https://user-images.githubusercontent.com/22256112/167352966-77280192-1b73-4bae-ac43-6ecc47a6dee3.png)